editor/plugins/version_control_editor_plugin.cpp

```diff
@@ -1145,14 +1146,18 @@ VersionControlEditorPlugin::VersionControlEditorPlugin() {
 	set_up_warning_text->set_h_size_flags(Control::SIZE_EXPAND_FILL);
 	set_up_settings_vbc->add_child(set_up_warning_text);
 
-	version_commit_dock = memnew(VBoxContainer);
+	version_commit_dock = memnew(EditorDock);
 	version_commit_dock->set_visible(false);
-	version_commit_dock->set_name(TTR("Commit"));
+	version_commit_dock->set_name(TTRC("Commit"));
+	version_commit_dock->set_dock_shortcut(ED_SHORTCUT_AND_COMMAND("docks/open_version_control", TTRC("Open Version Control Dock")));
```

Comment:
```suggestion
	version_commit_dock->set_name(TTRC("Commit"));
	version_commit_dock-> set_icon_name("VcsBranches");
	version_commit_dock->set_dock_shortcut(ED_SHORTCUT_AND_COMMAND("docks/open_version_control", TTRC("Open Version Control Dock")));
```
This isn't strictly necessary but the version control dock doesn't currently have an icon, so it will probably default to the "window" icon.